.github/workflows/prombench-gmp-collector.yml

```diff
@@ -36,9 +36,11 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: write
+      contents: read
```

Comment:
Why do you need these permissions?

Reply:
This is done to only have the necessary permissions for the `GITHUB_TOKEN`. The [default permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) are not required as the token is only used for commenting on a PR and reading docker images.
Ref: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token

Reply:
Let's add this section if end up having permission issues.